packages/language-server/src/test/prisma-fmt-referentialActions.test.ts

```diff
@@ -1,177 +0,0 @@
-import assert from 'assert'
```

Comment:
These tests don't exercise the language server completion code, but the prisma-fmt referential_actions() method, which wasn't used before this PR and becomes completely irrelevant with the prisma-fmt based completions. The relevant tests are in completion.test.ts.